### PR TITLE
Add netci.groovy for LLILC

### PR DESCRIPTION
**Pull request by adiaaida**
Our LLILC jobs should be generated by a groovy script like the rest of
the dotnet-ci jobs are. This change adds specifications for Windows
release and debug builds.

### CODE REVIEW

**Comment by adiaaida** [comment]
@mmitche ptal

**Comment by mmitche** [comment]
@dotnet-bot test ci please

File: netci.groovy [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+        branch('*/master')
+        relativeTargetDir('coreclr')
+      }
+    }
+  }
+}
+
+[true, false].each { isPR ->
+  ['Debug', 'Release'].each { configuration ->
+    def newJobName = Utilities.getFullJobName(project, configuration, isPR)
```
Comment (mmitche) [inline netci.groovy]: Change the second parameter if you want to add more info to change the generated job name:

e.g.

"windows_${configuration}"

File: netci.groovy [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+        relativeTargetDir('LLVM')
+      }
+      git {
+        remote {
+          github('dotnet/llilc')
+        }
+        branch('*/master')
+        relativeTargetDir('LLVM\\tools\\llilc')
+      }
+      git {
```
Comment (mmitche) [inline netci.groovy]: You want this one before the dotnet/llilc one

**Comment by adiaaida** [comment]
@dotnet-bot test ci please

**Pull request merged**